### PR TITLE
`package-lock.json` v3 breaking changes fixes

### PR DESCRIPTION
**Pull request by zamboney**
On `npm@9` the `package-lock` version change [form `v2` to `v3`](https://github.com/npm/cli/pull/5605). This change in `package-lock.json` v3 the `dependencies` property was remove.
This fix this issue by using the `packages` property and mapping the value to be like the `dependencies` property was before.

### CODE REVIEW

File: js/package.json [hunk 1]
```diff
@@ -1,32 +0,0 @@
...
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "husky": "^8.0.3",
-    "prettier": "^2.8.3",
-    "ts-node": "^10.9.1",
-    "ttypescript": "^1.5.15",
-    "typescript": "^4.9.4",
-    "yargs": "^17.6.2"
-  }
-}
```
Comment (kyryloonufriiev) [inline js/package.json:32]: Why?

Comment (zamboney) [reply js/package.json:32]: Mistake I will revert it

**Comment by SophieDePaula** [comment]
To make it a bit easier to look at this PR, notice that most changed files are just lint changes, the changes that are relevant here are:
`versions/index.js`
`versions/versions-utils.js`
`versions/index.spec.js`

**Comment by zamboney** [comment]
@SophieDePaula 10x for adding the missing `package.json`!
@kyryloonufriiev are we good for merge or you think there an issue I miss?

**Review by kyryloonufriiev — APPROVED** [review]

**Pull request merged**